include/battle.h

```diff
@@ -698,7 +698,10 @@ struct BattleStruct
     u8 magnitudeBasePower;
     u8 presentBasePower;
     u8 roostTypes[MAX_BATTLERS_COUNT][2];
-    u8 savedBattlerTarget;
+    u8 savedBattlerTarget[10];
+    u8 savedBattlerAttacker[10];
+    u8 savedTargetCount;
```

Comment:
Just to save some space I guess savedTargetCount and savedAttackerCount could be split into one byte with 4 bits. (This is optional though)

Also 10 bytes for each saved battler seems a bit overkill, I think 5 should be fine.